### PR TITLE
add cli skeleton

### PR DESCRIPTION
**Pull request by stoiev**
Merging full project in chunks through a feature branch

### CODE REVIEW

File: src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  def main(args: Array[String]): Unit = {
+
+    val opts = new TopicsMetadataUpsertCommandOptions(args)
+
+    if (args.length == 0)
+      CommandLineUtils.printUsageAndDie(opts.parser, "Diff or mirror topics between Kafkas.")
+
+    // should have exactly one action
+    val actions = Seq(opts.diffOpt, opts.mirrorOpt, opts.helpOpt).count(opts.options.has)
```
Comment (Evnsan) [inline src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala:23]: Could we call the application with --help option like this?
```
$CMD --diff --help
```
I usually call application like this in order to get command specific help.
But we can let it as an improvement to later versions too.

File: src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import java.util.Properties
+
+import scala.collection.JavaConverters._
+import scala.collection._
+
+import joptsimple.{ArgumentAcceptingOptionSpec, OptionParser, OptionSet, OptionSpec, OptionSpecBuilder}
+import kafka.utils.{CommandLineUtils, Logging}
+
+
+object TopicsMirrorCommand extends Logging {
```
Comment (Evnsan) [inline src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala:13]: Sorry, I can't get it. Does this class do something other than parse command args?

File: build.sbt [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import com.typesafe.sbt.packager.docker.DockerPermissionStrategy.CopyChown
+
+
+name := "kafka-topic-mirror"
+maintainer := "ivan.stoiev@cobli.co"
+organization := "co.cobli"
+
+// Let version come from CI branches/tags
+version := {
+  if(insideCI.value) {
```
Comment (Evnsan) [inline build.sbt:10]: From where is this variable `insideCI` is coming?

File: src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala (hunk 2 above)
Comment (stoiev) [reply src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala:13]: No. I'm just breaking PR's into a major feature branch. This is the first one with the cli skeleton.

File: build.sbt (hunk 3 above)
Comment (stoiev) [reply build.sbt:10]: It's a new sbt handy feature <3

File: src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala (hunk 1 above)
Comment (stoiev) [reply src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala:23]: `diff` and `mirror` has the same parameters. I think that we could eliminate both and implements a `--dry-run` option. What do you think?

Comment (Evnsan) [reply src/main/scala/co/cobli/kafka/mirror/TopicsMirrorCommand.scala:23]: Excellent idea!

**Pull request merged**